client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -104,10 +103,15 @@ const Header: FunctionComponent< Props > = ( { next, prev } ) => {
 			</div>
 			<div className="gutenboarding__header-section">
 				<div className="gutenboarding__header-group">
-					{ next && hasSelectedDesign && (
-						<Link to={ next } className="gutenboarding__header-next-button" isPrimary isLarge>
+					{ hasSelectedDesign && (
+						<Button
+							href={ `/start/frankenflow?theme=${ selectedDesign?.slug }&siteTitle=${ siteTitle }` }
```

Comment:
Frankenflow update to support this: https://github.com/Automattic/wp-calypso/pull/39070